addons/website_event_meet/views/event_meet_templates_page.xml

```diff
@@ -109,13 +110,20 @@
                         t-att-href="'/event/%s/meeting_room/%s' % (slug(event), slug(meeting_room_other))">
                         <div class="flex-grow-1 mw-100">
                             <div class="text-truncate" t-esc="meeting_room_other.name"/>
-                            <span class="text-muted" t-esc="meeting_room_other.summary"></span>
-                            <div class="d-flex justify-content-between align-items-center">
-                                <div class="text-muted">
-                                    <b>&amp;#9900;&amp;nbsp;</b>
-                                    <small><t t-esc="meeting_room_other.room_participant_count"/> <t t-esc="meeting_room_other.target_audience"/></small>
-                                </div>
-                                <small t-if="meeting_room_other.room_lang_id" class="text-muted"><i class="fa fa-globe"/> <t t-esc="meeting_room_other.room_lang_id.name"/></small>
+                            <div class="text-muted" t-esc="meeting_room_other.summary"/>
+                            <div class="text-muted" t-if="meeting_room_other.room_lang_id">
```

Comment:
The commit message says:
"[IMP] Event: Meeting room - Hide viewer count if no viewer".

But it seems to do a lot more than just that?
Can you elaborate?

Reply:
I put the unrelated changes to another commit.